backend/apis/parse-docket-pdf.utils.test.js

```diff
@@ -0,0 +1,62 @@
+const { file } = require("./docket-pdf-test-files/file");
+const { parsePdfText } = require("./parse-docket-pdf.utils");
```

Comment:
So I think that `parse-docket-pdf.utils.test.js` should maybe be renamed to `normal-docket-pdf.test.js`. The idea is that we are going to have many different docket pdfs that have their own tests, each one verifying different edge cases with the PDFs. So we want the file names to not be as generic.